spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationExecution.scala

```diff
@@ -108,16 +102,17 @@ trait StandardizationExecution extends CommonJobExecution with S3DefaultCredenti
     }
   }
 
-  override def validateOutputPath(s3Config: S3Config, pathConfig: PathConfig)(implicit fsUtils: DistributedFsUtils): Unit = {
-    validateIfPathAlreadyExists(s3Config, pathConfig.standardizationPath)
+  override def validateOutputPath(pathConfig: PathConfig)(implicit fileSystems: FileSystems): Unit = {
+    // Std output is validated in the std FS
+    validateIfPathAlreadyExists(pathConfig.standardizationPath)(fileSystems.standardizationFs.toFsUtils)
   }
 
   protected def readStandardizationInputData[T](schema: StructType,
                                                 cmd: StandardizationConfigParser[T],
                                                 path: String,
                                                 dataset: Dataset)
                                                (implicit spark: SparkSession,
-                                                fsUtils: DistributedFsUtils,
+                                                rawFs: FileSystem,
```

Comment:
Not sure if this should be kept as an implicit

Reply:
Specifically here or in general for FileSystem? I thought about this as well. In cases where only a single FS is used, I think it could. 

With multiple fs, it makes no sense with the current types (I thought about introducing something like `case class InputFs(fs: FileSystem)` and `outputFs` in a similar fashion, then you could have these as implicits and create implicits as needed (because `stdFs` is sometimes an `inputFs` and sometimes an `outputFs`), but I have abandoned this idea because it seemed that it is not worth doing for the sake of 'using implicits', for those cases just explicitly passing a specific FS seems more appropriate.

I will, nonetheless, review the implicit FS usage, your point makes sense.

Reply:
Here, the implicit usage seems justified (there no filesystems around to get confused with). Or are you suggesting to use explicit filesystems in Enceladus everywhere?

Reply:
I put my comment on a wrong line. My main concern was `PerformanceMetricTools.addJobInfoToAtumMetadata`

Reply:
Oh, you were spot on. I have redone the API of that with relation the FileSystems+PathConfig merge. The implicit is now gone.